src/EPR.Payment.Service.UnitTests/Services/RegistrationFees/ProducerFeesCalculatorServiceTests.cs

```diff
@@ -201,6 +210,7 @@ public async Task CalculateFeesAsync_WhenLargeProducerWith50SubsidiariesWithProd
             {
                 result.ProducerRegistrationFee.Should().Be(262000m); // £2,620 represented in pence
                 result.ProducerOnlineMarketPlaceFee.Should().Be(257900m); // Online Market fee in pence
+                result.ProducerLateRegistrationFee.Should().Be(0m); // Late fee in pence
```

Comment:
I think it is worth adding IsLateFeeApplicable = false in ProducerRegistrationFeesRequestDto, just for clarity.